webrtc.html

```diff
@@ -3227,6 +3227,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           not reset to zero once the channel closes).</p>
         </dd>
 
+        <dt>attribute unsigned long lowBufferLimit</dt>
+
+        <dd>
+          <p>The <dfn id=
+          "dom-datachannel-lowbufferlimit"><code>lowBufferLimit</code></dfn>
+          attribute sets the threshold at which the <code><a href=
+          "#dom-datachannel-bufferedamount">bufferedAmount</a></code> is
+          considered to be low.  When the <code><a href=
+          "#dom-datachannel-bufferedamount">bufferedAmount</a></code>
+          decreases from above this threshold to equal or below it, the
+          <code title="event-RTCDataChannel-bufferlow"><a href=
+          "#event-datachannel-bufferlow">bufferlow</a></code> event fires.
+          The <code><a href=
+          "#dom-datachannel-lowbufferlimit">lowBufferLimit</a></code>
+          defaults to zero, but the application may change its value at any
+          time.</p>
```

Comment:
I think that you want to say that this value starts at zero rather than defaults.